Dockerfile

```diff
@@ -35,3 +35,13 @@ EXPOSE ${WEBAPP_PORT}
 
 # DEV - ENTRYPOINT/CMD
 ENTRYPOINT ["poetry", "run", "flask", "run", "--host", "0.0.0.0"]
+
+# TEST CONTAINER==========================================================================
+FROM base as test
+
+# TEST - EXPOSE PORT 
+ENV WEBAPP_PORT=6000
+EXPOSE ${WEBAPP_PORT}
```

Comment:
Note that we don't really need a port exposed for our test app - you never use it in your `docker run ...` commands, because the tests all happen internally within the container, so we may as well remove these lines